### PR TITLE
Selenium: Inspect methods logic in the "CommandsEditor" pageobject

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Inspect methods logic in the "CommandsEditor" pageobject

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/9169

### CODE REVIEW

**Comment by Ohrimenko1988** [comment]
ci-test

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 1]
```diff
@@ -32,24 +36,21 @@
...
 import org.openqa.selenium.Keys;
-import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedCondition;
-import org.openqa.selenium.support.ui.WebDriverWait;
 
-/** @author Aleksandr Shmaraiev */
+/** @author Ihor Okhrimenko */
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: Aleksandr Shmaraiev is still one of the author.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java [hunk 2]
```diff
@@ -423,33 +415,36 @@ public String getAllVisibleTextFromMacrosContainer() {
...
+        .until(
+            (ExpectedCondition<Boolean>)
+                driver -> {
+                  String backgroundColor =
+                      waitVisibilityAndGetMacrosCommand(item).getCssValue("background-color");
+                  return backgroundColor.contains("rgba(215, 215, 215, 0.2)");
+                });
+  }
+
+  public WebElement waitVisibilityAndGetMacrosCommand(String item) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java]: `MacrosCommand` sounds misleading.
Let's name it "CommandMacro" or just "Macro" due to [Eclipse Che documentation](https://www.eclipse.org/che/docs/ide/commands/index.html#macros): `waitVisibilityAndGetMacro()`

**Comment by dmytro-ndp** [comment]
I would recommend to fix misleading term `MacroCommand` on just `Macro`, due to [Eclipse Che terminology](https://www.eclipse.org/che/docs/ide/commands/index.html#macros).

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java [hunk 3]
```diff
@@ -101,7 +101,6 @@ public void checkCommandMacrosIntoCommandLine() {
     commandsEditor.selectMacroCommand("${current.class.fqn}");
     commandsEditor.typeTextIntoEditor(Keys.ARROW_DOWN.toString());
     commandsEditor.typeTextIntoEditor(Keys.SPACE.toString());
-    commandsEditor.waitMacroCommandIsSelected("${current.project.path}");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java:104]: Why did you decide to remove this verification?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java [hunk 4]
```diff
@@ -142,7 +141,6 @@ public void checkCommandMacrosIntoPreviewUrl() {
     commandsEditor.selectMacroCommand("${server.wsagent/http}");
     commandsEditor.typeTextIntoEditor(Keys.ARROW_UP.toString());
     commandsEditor.typeTextIntoEditor(Keys.SPACE.toString());
-    commandsEditor.waitMacroCommandIsSelected("${server.wsagent/http}");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java:145]: Why did you decide to remove this verification?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java (hunk 3 above)
Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java:104]: It hard to check a selection now

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java (hunk 4 above)
Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java:145]: It hard to check a selection now

**Review by dmytro-ndp — APPROVED** [review]

**Comment by dmytro-ndp** [comment]
@Ohrimenko1988: by the way: it seems we don't need `seleniumWebDriver` field in **CommandEditor** class.

**Comment by Ohrimenko1988** [comment]
@dmytro-ndp  no we steal need it :
```
 public void deleteAllContent() {
    actionsFactory
        .createAction(seleniumWebDriver)
        .keyDown(CONTROL)
        .sendKeys("a")
        .keyUp(CONTROL)
        .sendKeys(DELETE)
        .perform();
  }
```

**Comment by Ohrimenko1988** [comment]
ci-test

**Comment by Ohrimenko1988** [comment]
ci-test

**Review by SkorikSergey — APPROVED** [review]

**Pull request merged**